sale_documents_comments/models/stock.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# © 2015 Ainara Galdona - AvanzOSC
+# © 2015 Oihane Crucelaegui - AvanzOSC
+# © 2015 Esther Martín <esthermartin@avanzosc.es> - Avanzosc S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, models, fields
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    sale_comment = fields.Text(string='Internal comments')
+    sale_propagated_comment = fields.Text(
+        string='Propagated internal comments')
+
+    @api.one
+    @api.onchange('partner_id')
+    def onchange_partner_id(self):
+        picking_com, picking_pcom = self.partner_id._get_picking_comments()
+        self.sale_comment = picking_com
+        self.sale_propagated_comment = picking_pcom
+
+    @api.model
+    def create(self, values):
```

Comment:
you get very weird errors in a mixed old / new API overloading scenario. Say someone creates a new addon with a model with `_inherit` from yours and writes it with the old api. He may call `super(...).create(...)` and not get an integer as return value, which will very likely cause a crash. This can be safely removed in v10.